README.md

```diff
@@ -431,9 +431,10 @@ bundle exec rubocop # Runs the lint (use `--fix` for autocorrect)
 
 ## Releasing a new gem version
 
+1. From updated master: `git checkout -b bump-vX.X.X`, according to [SemVer](https://semver.org)
 1. Ensure the CHANGELOG is correct and updated, this is your last opportunity
-2. Execute `bundle exec bump patch --tag`, or minor or major according to [SemVer](https://semver.org), this bumps the version in your local machine in the latest commit
-3. Push to GitHub `git push && git push origin vX.X.X`
+2. Execute `bundle exec bump patch --tag`, or minor or major, this bumps the version in your local machine in the latest commit
+3. Push to GitHub `git pull && git push origin vX.X.X`
```

Comment:
The process is still a bit manual for release candidates, but it's not too hard.